ehsm_kms_service/payload_checker.js

```diff
@@ -49,7 +49,7 @@ const keyid = {
 }
 const aad = {
   type: PARAM_DATA_TYPE.BASE64,
-  maxLength: MAX_LENGTH,
+  maxLength: MAX_LENGTH*4/3,

```

Comment:
does it enough? and also please add comments to explain it's the real size after base64 encoding